ocaml/lib/snarky_js_bindings_lib.ml

```diff
@@ -1,58 +1,12 @@
+open Core_kernel
```

Comment:
Nice! Should we name this file something like `snarky_bindings.ml` then? wdyt

Reply:
Good idea! How do you like it now? I also removed the top level `Snarky` module in `snarky_bindings.ml` now because there's nothing else in that file